packages/quantic/force-app/main/default/lwc/quanticFeedbackModal/__tests__/quanticFeedbackModal.test.js

```diff
@@ -0,0 +1,436 @@
+      it('should display and log an error message', async () => {
+        const element = createTestComponent({...defaultOptions, options: []});
+        await flushPromises();
+
+        const errorMessage = element.shadowRoot.querySelector(
+          selectors.errorMessage
+        );
+
+        expect(errorMessage).not.toBeNull();
+        expect(console.error).toHaveBeenCalledWith(errors.emptyArrayError);
+      });
+    });
+
+    describe('when one of the options does not contain a label', () => {
+      it('should display and log an error message', async () => {
+        const element = createTestComponent({
+          ...defaultOptions,
+          options: [{value: exampleValue}],
+        });
+        await flushPromises();
+
+        const errorMessage = element.shadowRoot.querySelector(
+          selectors.errorMessage
+        );
+
+        expect(errorMessage).not.toBeNull();
+        expect(console.error).toHaveBeenCalledWith(
+          errors.missingLabelOrValueError
+        );
+      });
+    });
+
+    describe('when one of the options does not contain a value', () => {
+      it('should display and log an error message', async () => {
+        const element = createTestComponent({
+          ...defaultOptions,
+          options: [{label: exampleValue}],
+        });
+        await flushPromises();
+
+        const errorMessage = element.shadowRoot.querySelector(
+          selectors.errorMessage
+        );
+
+        expect(errorMessage).not.toBeNull();
+        expect(console.error).toHaveBeenCalledWith(
+          errors.missingLabelOrValueError
+        );
+      });
+    });
+
+    describe('when a label of one of the options is not of type string', () => {
+      it('should display and log an error message', async () => {
+        const invalidLabel = 1;
+        const element = createTestComponent({
+          ...defaultOptions,
+          options: [{label: invalidLabel, value: exampleValue}],
+        });
+        await flushPromises();
+
+        const errorMessage = element.shadowRoot.querySelector(
+          selectors.errorMessage
+        );
+
+        expect(errorMessage).not.toBeNull();
+        expect(console.error).toHaveBeenCalledWith(
+          errors.invalidLabelTypeError(invalidLabel)
+        );
+      });
+    });
+
+    describe('when a value of one of the options is not of type string', () => {
+      it('should display and log an error message', async () => {
+        const invalidValue = 1;
+        const element = createTestComponent({
+          ...defaultOptions,
+          options: [{label: exampleValue, value: invalidValue}],
+        });
+        await flushPromises();
+
+        const errorMessage = element.shadowRoot.querySelector(
+          selectors.errorMessage
+        );
+
+        expect(errorMessage).not.toBeNull();
+        expect(console.error).toHaveBeenCalledWith(
+          errors.invalidValueTypeError(invalidValue)
+        );
+      });
+    });
+  });
```

Comment:
Does it really makes sense to validate all of that AND have to write tests for it?

Reply:
On one hand salesforce error messages often leave more to be desired, on the other I think you're right that it does add a considerable amount of code and I think any code that can be validated with tests should.
I'm all for improving the developer experience by adding informative error messages but only to prevent truly breaking options so there's a balance to strike. My understanding is that Atomic does a lot of prop validation and it would be unfortunate if Quantic didn't try to deliver a similar quality of dev experience.

Anyway in this particular case, the code and the tests are already written so if the validations are limited to ensuring the component doesn't crash I'm for it.

Reply:
As you may have already noticed in my PR, I'm all for improving the developer experience by adding this type of validation.
The main reason is that this will help avoid uncertain/unknown behaviour with Headless down the road.
For example in this component, the value of a given option is passed  down the road to a function of one of the Headless controllers, if this function accepts only a parameter of type string and throws an error otherwise, it wouldn't be easy to recognize the source of the issue just by looking at the error logged by Headless. However if we handle this prop validation in Quantic the user can solve the problem in no time while developing his project.
That's the value I see in making this kind of validations and frankly, this seems useful to me.
What do you think @erocheleau ?